RFCs/use-prs-instead-of-issues.md

```diff
@@ -0,0 +1,8 @@
+# Use PRs instead of issues for RFCs
+
+We already have a [playbook](../playbooks/rfcs.md) for RFCs. We used to use GitHub [issues](https://github.com/artsy/README/issues?q=is%3Aissue+is%3Aclosed), but this approach has two drawbacks: 
+
+- it's impossible to add inline or threaded comments to an issue, and the conversation can become very hard to follow
```

Comment:
using this worked

> wow

but yea, that totally makes sense, cause now we can have many threads running!